TypeScript/src/components/Order.ts

```diff
@@ -0,0 +1,120 @@
+type MaybeNumber = number | null;
+
+export class Order{
+    bagsNumber:MaybeNumber;
+    private readonly coffePrice:MaybeNumber;
+    private readonly largeBoxPrice:MaybeNumber;
+    private readonly mediumBoxPrice:MaybeNumber;
+    private readonly smallBoxPrice:MaybeNumber;
+    private readonly large:MaybeNumber;
+    private readonly medium:MaybeNumber;
+    private readonly small:MaybeNumber;
+    private readonly totalPricePercentageDiscount:MaybeNumber;
+    private readonly discountCostRange:MaybeNumber;
+    largeUsed:MaybeNumber;
+    mediumUsed:MaybeNumber;
+    smallUsed:MaybeNumber;
+    rest:MaybeNumber;
+    totalCoffePrice:MaybeNumber;
+    totalLargeBoxesPrice:MaybeNumber;
+    totalMediumBoxesPrice:MaybeNumber;
+    totalSmallBoxesPrice:MaybeNumber;
+    totalPriceInitial:MaybeNumber;
+    totalPriceFinal:MaybeNumber;
+
+    orginizeBags(): void {
```

Comment:
can you explain in english what this means? 
orginizeBags(): void

Reply:
orginizeBags is the function that is responsible for distributing the coffe bags in Large, medium and small bags

Reply:
It is void because it won't return a string array or anything else. It is responsible for setting the parameters in the object created